specs/experimental/zk-fault-dipute-game/overview.md

```diff
@@ -13,14 +13,88 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+[g-withdrawal]: ../../glossary.md#withdrawal
+
 ## Overview
 
+The ZK Fault Dispute Game (ZKFDG) is a new type of dispute game compatible with the OP Stack. In ZKFDG, the first
+disagreeing output root is found by the process called "dissection". Then, both derivation of L2 block from L1
+corresponding to that output root and executing the block's transactions is executed in zkVM. The validity of the
+execution is guaranteed by the ZK proof, which can be verified on-chain.
+
+The main difference from [Kroma's previous ZK Fault Proof System](../../fault-proof/challenge.md) is that ZKFDG uses a
+"zkVM" instead of a "zkEVM" to handle all processes. By using zkVM for fault proof, it is enabled to verify all
```

Comment:
```suggestion
"zkVM" instead of a "zkEVM" to handle the proving fault process. By using zkVM for fault proof, it is possible to verify the entire
```